packages/neutrino/src/api.js

```diff
@@ -26,14 +26,26 @@ const pathOptions = [
 
 // getOptions :: Object? -> IO Object
 const getOptions = (opts = {}) => {
+  let extensions = new Set(['js', 'jsx', 'vue', 'ts', 'mjs']);
   const options = merge({
     env: {
       NODE_ENV: 'development'
     },
     debug: false,
-    quiet: false
+    quiet: false,
+    extensions
   }, opts);
 
+  Object.defineProperty(options, 'extensions', {
```

Comment:
@eliperelman Let me know if this is the way you wanted it to be done. Once we've agreed on a way, I could update the rest neutrino to use it.

Reply:
As discussed in IRC, this can just be a normal property, and consumers can access the Set directly. We can also expose a method for generating a regex on the fly, but we don't want to limit this property to only creating regexes.

Reply:
Another thing, a user should be able to pass this as an array to Neutrino `options`, which when merged using `mergeOptions` or whatever, can be merged with the Set values:

```js
module.exports = {
  options: {
    extensions: ['vue']
  }
};

// ...

new Set([...this.options.extensions, ...extensions])
```

When merging we should also probably normalize the values to ensure they don't contain dots.